apps/docs/docs/tutorials/dashboard/manage.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 4
+title: Manage groups
+---
+
+# Manage groups
```

Comment:
The first time that the Client app is mentioned, it would be nice to let people know that the client app is like a demo app to test those things but it's better if they create their own app to add people to a group using an invite code and to generate the access link for credential groups.

Reply:
In this case, do you think it will be beneficial to create a tutorial section for Client app too? Or just redirect them to look at the Client app source code?

Reply:
Hey @waddaboo! I don't think so. Just mentioning what the client app is and maybe just adding the code to add a member using an invite code is fine. When explaining the invite code, it would be nice to also say that the invite code can be an invite link with a custom url if they create an app that has a page that adds a member when using an invite code. 

Structure:

```bash
https://<custom-domain>?inviteCode=<invite-code>
```

Example:

```bash
https://client.bandada.pse.dev?inviteCode=MQYS4UR5
```

They don't have to use that structure for their url, but that's an example.